maxim/logger/pydantic_ai/client.py

```
@@ -0,0 +1,737 @@
+"""Fixed Maxim integration for Pydantic AI agent framework."""
+
+import contextvars
+import functools
+import importlib
```

Ruff (0.12.2)

`importlib` imported but unused

Remove unused import: `importlib`

(F401)